Microsoft.Toolkit.Uwp.SampleApp/SamplePages/SlidableListItem/SlidableListItemCode.bind

```diff
@@ -27,7 +28,8 @@
                     IsOffsetLimited="@[IsOffsetLimited:Bool:true]"
                     MouseSlidingEnabled="@[MouseSlidingEnabled:Bool:true]"
                     LeftCommand="{x:Bind ToggleFavorite}"
-                    RightCommandRequested="SlidableListItem_RightCommandActivated">
+                    RightCommand="{Binding DeleteItem, ElementName=Page, Mode=OneWay}"
```

Comment:
Just use x:Bind here rather than binding, means you don't have to name the page.